Makefile

```diff
@@ -79,3 +79,8 @@ fauxton: share/www
 share/www:
 	@echo "Building Fauxton"
 	@cd src/fauxton && npm install && ./node_modules/grunt-cli/bin/grunt couchdb
+
+update: clean
+	@git pull https://git-wip-us.apache.org/repos/asf/couchdb.git master
```

Comment:
May be make repourl configurable via env variable and the branch as well? Then, you can easily changes them if you're need to use your fork or specific branch like developers-preview or release tag.

Something like:
`make update repo=https://github.com/robertkowalski/couchdb ref=12345-fix-very-nasty-issue`